src/main/java/Duke.java

```diff
@@ -1,10 +1,179 @@
+                break;
+            case "mark":
+                int taskIndex = Integer.parseInt(inputs[1]);
+                if (taskIndex > tasks.size() || taskIndex <= 0) throw  new IllegalInputException();
+                tasks.get(taskIndex-1).markAsDone();
+                System.out.println("I have marked this task as done");
+                printList(taskCounter);
+                break;
+            case "unmark":
+                taskIndex = Integer.parseInt(inputs[1]);
+                if (taskIndex > tasks.size() || taskIndex <= 0) throw  new IllegalInputException();
+                tasks.get(taskIndex-1).markAsNotDone();
+                System.out.println("I have unmarked this task");
+                printList(taskCounter);
+                break;
+            case "todo":
+                inputTodo(inputs, taskCounter);
+                break;
+            case "deadline":
+                String deadlineInput = "";
+                String deadlineDeadline = "";
+                boolean isDeadline = false;
+                for (int i = 1; i < inputs.length; i ++) {
+                    if (inputs[i].contains("/")) {
+                        isDeadline = true;
+                        continue;
+                    }
+                    if (isDeadline) {
+                        deadlineDeadline += inputs[i];
+                        deadlineDeadline += " ";
+                    }
+                    else {
+                        deadlineInput += inputs[i];
+                        deadlineInput += " ";
+                    }
+                }
+                if (!isDeadline) throw new IllegalInputException();
+                Deadline deadline = new Deadline(deadlineInput, deadlineDeadline);
+                tasks.add(deadline); taskCounter += 1;
+                printAddDeadline(deadline, taskCounter);
+                break;
+            case "event":
+                String eventInput = "";
+                String eventFrom = ""; String eventTo = "";
+                boolean isFrom = false; boolean isTo = false;
+                for (int i = 1; i < inputs.length; i ++) {
+                    if (inputs[i].contains("/from")) {
+                        isFrom = true;
+                        continue;
+                    }
+                    if (inputs[i].contains("/to")) {
+                        isTo = true;
+                        continue;
+                    }
+                    if (isTo) {
+                        eventTo += inputs[i];
+                        eventTo += " ";
+                    }
+                    else if (isFrom){
+                        eventFrom += inputs[i];
+                        eventFrom += " ";
+                    }
+                    else {
+                        eventInput += inputs[i];
+                        eventInput += " ";
+                    }
+                }
+                if (! (isTo && isFrom)) throw new IllegalInputException();
```

Comment:
Coding Standard: Single statement conditionals should still be wrapped by curly brackets.


Remember to add the braces even if it is a single line instruction and the syntax allows it, to follow the coding standard.